compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -833,12 +833,18 @@ class Typer extends Namer
       case _: WildcardType => untpd.TypeTree()
       case _ => untpd.TypeTree(tp)
     }
+    def interpolateWildcards = new TypeMap {
+      def apply(t: Type) = t match
+        case WildcardType => newTypeVar(TypeBounds.empty)
```

Comment:
`newTypeVar` will call `constrained(poly, untpd.EmptyTree, ...)` but we do actually have an owning tree that we could use here I think.